docs/decision-records/DR-0002-dr-validation.md

```diff
@@ -0,0 +1,86 @@
+If an only if, the **Corresponding Author** is not a Collaborator, they MUST find a Sponsor.
+A **Sponsor** is a Collaborator who shepherds the decision record through the validation process and makes sure:
+- (during the validation process / PR Review) that the **Corresponding Author** is actually heard, the proposal considered, and the process respected
+- (after the decision has been recorded / PR merged) the **decision** is applied, respected and understood (i.e. if someone has a question about this **decision**, they should reach to the **Sponsor**)
+
+The **Sponsor**, and the **Corresponding Author** SHOULD be clearly defined in a markdown header of the **Decision Recording** following the decision framework templates.
+
+A Pull Request needs Reviewers and an Assignee.
+The **Corresponding Author** and the **Sponsor** are not eligible for those roles. Anyone else is eligible.
+
+Anyone MAY, and at least one of the {**Corresponding Author**, **Sponsor**} MUST designate one or more eligible Reviewers.
+An eligible Reviewer MAY designate themselves.
+
+The **Corresponding Author** OR the **Sponsor** MUST assign the **Decision Recording** Pull Request to an eligible Assignee.
+The Assignee MAY also be a Reviewer.
+
+The Assignee looks for consensus before merging the **Decision Recording** Pull Request.
+In clear absence of consensus the Assignee closes the request.
+It is the responsibility of the Assignee to act and not let PR remain at a stale point.
+
+Whenever possible (unless they are not eligible, or for other reasons known to be unavailable), the Assignee SHOULD be:
+
+1. `@noelmace` for everything related to communication, openness framework, DevX and DevRel in general
+2. `@filipblnt` for everything related to engineering, by default (adding a member of the engineering team as an additional Reviewer whenever needed)
+3. `@milstan` for everything related to our company's "vision" and high-level concepts (i.e. high-level product ownership)
+4. `@Vuk-BN` for everything related to global executive strategies and finances
+
+When such assignment is not possible, the then the Assignee SHOULD be the first eligible and available person from the above list, in the order from 1. to 4.
+
+### Validation protocol for non template-based documents
+
+>This protocol applies, unless a particular project-related workflow is specified in the project's readme file, when producing other types of documents that MAY also involve more or less normative **choices** and even certain **decisions** (e.g. user stories, specifications, designs, documentation work, code) but that do not use one of the [Decision Framework's specific document templates](https://github.com/blindnet-io/openness-framework/tree/main/DecisionFramework#what-template-will-you-use)
+
+One of the Authors, the **Corresponding Author**, makes a PR.
+They invite Reviewers others then themselves.
+A Reviewer MAY designate themselves.
```

Comment:
Reading this, I think it could be unclear _to what_ and _how_ a rewiewer may designate themselves.

The following might be a better phrasing:
```suggestion
Anyone MAY [review a PR](https://github.com/blindnet-io/openness-framework/blob/main/docs/HOWTOs/PR-review.md), therefore designating themselves as a **Reviewer**.
```